build.sh

```diff
@@ -9,10 +9,10 @@ Usage: ./build.sh [options]
 
 Possible options:
   -h,         --help                  Display help
+              --deps-only             Only install dependencies (don't build)
```

Comment:
```suggestion
  -o,         --deps-only             Install dependencies only (don't build)
```

1) Let's add short option alias for uniformity with other options

2) Placing "only" at the end aligns with natural English phrasing. Or we can write something like "Install dependencies without building".

3) And don't forget to sync description with description on line 29.

Reply:
I didn't want to add the short option o because in many unix tools it is fixed for output and can cause confusion